### PR TITLE
Add mercurial script

### PR DESCRIPTION
**Pull request by pele1410**
I added support for tracking mercurial repositories similar to the existing git repository support.

```
set -g @dracula-plugins "hg time"
```

Clean repository
clean

Dirty repository
dirty

Tested each configuration:
```
set -g @dracula-hg-disable-status true
set -g @dracula-hg-show-diff-symbol "^"
```
hg-disable-status

```
set -g @dracula-hg-no-untracked-files true
```
no-untracked-files

```
set -g @dracula-hg-no-repo-message "no-hg"
```
hg-no-repo-message

```
set -g @dracula-hg-show-current-symbol "@"
```
hg-show-current-symbol

### CODE REVIEW

File: scripts/hg.sh [hunk 1]
```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $current_dir/utils.sh
+
+IFS=' ' read -r -a hide_status <<< $(get_tmux_option "@dracula-hg-disable-status" "false")
+IFS=' ' read -r -a current_symbol <<< $(get_tmux_option "@dracula-hg-show-current-symbol" "✓")
+IFS=' ' read -r -a diff_symbol <<< $(get_tmux_option "@dracula-hg-show-diff-symbol" "!")
+IFS=' ' read -r -a no_repo_message <<< $(get_tmux_option "@dracula-hg-no-repo-message" "")
+IFS=' ' read -r -a no_untracked_files <<< $(get_tmux_option "@dracula-hg-no-untracked-files" "false")
```
Comment (ethancedwards8) [inline scripts/hg.sh:10]: Can you document these in INSTALL.md?

**Review by ethancedwards8 — CHANGES_REQUESTED** [review]

File: scripts/hg.sh (hunk 1 above)
Comment (pele1410) [reply scripts/hg.sh:10]: No problem, thought I had them all

Comment (pele1410) [reply scripts/hg.sh:10]: Updated

**Review by ethancedwards8 — APPROVED** [review]

**Comment by ethancedwards8** [comment]
Thank you for your contribution!

**Pull request merged**